docker-compose.yml

```
@@ -1,23 +1,29 @@
 redis:
   build: ./redis
+  expose:
+    - "6379"
   ports:
     - "6379:6379"
-  net: "host"
   env_file: .env
   container_name: denguetorpedo-redis
   volumes:
-    - <local_folder_for_redis>:/bitnami
+    - /Users/Alan/Documents/UCA-CYT/SE2/2019/denguechat-compose/redis/redisVolume:/bitnami
```

Keep it as it was, with a variable name rather than your actual local folder (which will not be the same in other systems)
```suggestion
    - <local_folder_for_redis>:/bitnami
```